frontend/packages/dev-console/src/components/gitops/GitOpsEmptyState.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { EmptyState, EmptyStateIcon, EmptyStateVariant } from '@patternfly/react-core';
+import * as gitopsIcon from '../../images/gitops.svg';
+import './GitOpsEmptyState.scss';
+
+interface GitOpsEmptyStateProps {
+  emptyStateMsg: string;
+}
+
+const gitopsImage = () => (
+  <img className="odc-gitops-empty-state__icon" src={gitopsIcon} alt="GitOps" />
+);
+
+const GitOpsEmptyState: React.FC<GitOpsEmptyStateProps> = ({ emptyStateMsg }) => (
+  <EmptyState variant={EmptyStateVariant.full}>
+    <p className="odc-gitops-empty-state__msg">{emptyStateMsg}</p>
+    <EmptyStateIcon variant="container" component={gitopsImage} />
+  </EmptyState>
```

Comment:
Better to use PF standard way of rendering empty state.
```suggestion
  <EmptyState variant={EmptyStateVariant.full}>
    <EmptyStateIcon variant="container" component={gitopsImage} />
    <EmptyStateBody>{emptyStateMsg}</EmptyStateBody>
  </EmptyState>
```

Reply:
I was trying keep it similar to what we have for helm empty-state https://github.com/openshift/console/pull/6019#pullrequestreview-450723548.